static/js/src/advantage/auto-renewal.js

```diff
@@ -22,6 +22,13 @@ function confirmChanges() {
 
   setAutoRenewal(enabledRadio.checked)
     .then((data) => {
+      dataLayer.push({
+        event: "GAEvent",
+        eventCategory: "Advantage",
+        eventAction: "set-auto-renewal",
+        eventLabel: "Save changes",
+        eventValue: enabledRadio.checked,
```

Comment:
The value field should always be undefined throughout these events. The value needs to be a positive number that is added up to help GA goals in analytics which we don't have set up. https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventValue